zproject/backends.py

```diff
@@ -319,6 +321,19 @@ def ldap_to_django_username(self, username: str) -> str:
         return username
 
 class ZulipLDAPAuthBackend(ZulipLDAPAuthBackendBase):
+    fake_ldap_auth_enabled = getattr(settings, 'FAKE_LDAP_AUTH_ENABLED', False)
```

Comment:
We should put a default value of `False` in `DEFAULT_SETTINGS`.  As a policy rule, we do that consistently to avoid needing to use `getattr` to interact with settings.

(In case you haven't seen it, see http://zulip.readthedocs.io/en/latest/subsystems/settings.html)